platforms/core-configuration/isolated-action-services/src/main/kotlin/org/gradle/internal/isolate/actions/services/DefaultIsolatedProjectEvaluationListenerProvider.kt

```diff
@@ -42,79 +42,6 @@ private
 typealias IsolatedProjectActionList = Collection<IsolatedProjectAction>
 
 
-internal
```

Comment:
❓ 

Is this just a move (to line 147), or was there any code that changed as well?

Reply:
It seems the IDE `Move type` refactoring caused the types to be in a different order.

Reply:
This has been fixed, thanks for the review @abstratt!